### ISSUE TITLE
[ASUS Z87M-PLUS/Intel graphics] GUI freezing and being unresponsive

### ISSUE THREAD

**Issue by dreamawakening** [pop-os/pop#1960]
**Distribution (run `cat /etc/os-release`):** 

```
NAME="Pop!_OS"
VERSION="21.04"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.04"
VERSION_ID="21.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=hirsute
UBUNTU_CODENAME=hirsute
LOGO=distributor-logo-pop-os
```
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):** GNOME 3.38.5

**Issue/Bug Description:**

The GUI keeps freezing and being unresponsive, to the point my OS is almost unusable. 

I can't click on any applications. I can only do a few things when this happens, such as switching workspaces.

Restarting the GNOME shell hasn't made this problem go away, nor has rebooting.

This started happening over the past day or so. I haven't changed anything in that time, besides updating packages in the Pop!_Shop (which includes OS-related components).

I haven't tweaked anything or done anything I'm aware of that would've caused this bug.

**Expected behavior:**

It should work without freezing - especially after a fresh install of a stable OS version. I only installed Pop!_OS about 3 weeks ago - it worked fine up until recently.

**Other Notes:**

I've never had this problem with Ubuntu, and I'm considering switching back if this is what Pop!_OS is going to be like.

**Comment by dspacejs** [comment]
@jacobgkau 

CPU: Intel® Core™ i5-4670K CPU @ 3.40GHz × 4 
Graphics: Intel® HD Graphics 4600 (HSW GT2)
Motherboard: ASUS (IIRC their Z87M-PLUS model)

**Comment by dspacejs** [comment]
Nope, after rebooting (without opening Tauon) this bug happened again. @jacobgkau

**Comment by dspacejs** [comment]
@jacobgkau is there some way for me to fix this? It's honestly a huge nuisance and I'm considering switching back to Ubuntu because of it (which I'd rather not).
